### PR TITLE
Add bounding boxes to icons

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13613
affects: Student, Teacher, Parent
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student, Teacher, Parent

[MBL-13613](https://instructure.atlassian.net/browse/MBL-13613)

<details><summary>:x: XCTest failed: StudentUnitTests/ActivityStreamViewControllerTests/testLayout</summary>


    Assertion Failure: ActivityStreamViewControllerTests.swift:61: XCTAssertEqual failed: ("Optional("Nov 20")") is not equal to ("Optional("Nov 19")")
    Assertion Failure: ActivityStreamViewControllerTests.swift:67: XCTAssertEqual failed: ("Optional("Nov 19")") is not equal to ("Optional("Nov 18")")
    Assertion Failure: ActivityStreamViewControllerTests.swift:73: XCTAssertEqual failed: ("Optional("Nov 18")") is not equal to ("Optional("Nov 17")")
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1012802&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|3
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|3

**Comment by vanwagonet** [comment]
This won't fix every single icon everywhere, but it fixes a lot of them. And going forward we have icons with the correct bounding box, so we can fix more as we determine we need to.

File: Student/Student/Notifications/ActivityStreamViewController.swift [hunk 1]
```diff
@@ -86,7 +86,7 @@ class ActivityStreamViewController: UIViewController, PageViewEventViewControlle
         let refresh = UIRefreshControl()
         refresh.addTarget(self, action: #selector(refresh(_:)), for: .valueChanged)
         tableView?.refreshControl = refresh
-        tableView.tableFooterView = UIView()
```
Comment (vanwagonet) [inline Student/Student/Notifications/ActivityStreamViewController.swift]: this was moved to the storyboard.

File: rn/Teacher/src/canvas-api-v2/queries/ContextCard.js [hunk 2]
```diff
@@ -97,6 +97,7 @@ export default gql`query StudentContextCard($courseID: ID!, $userID: ID!) {
               html_url: htmlUrl
               points_possible: pointsPossible
               grading_type: gradingType
+              submission_types: submissionTypes
```
Comment (vanwagonet) [inline rn/Teacher/src/canvas-api-v2/queries/ContextCard.js]: need submission types to get correct icon for listed assignments.

**Pull request merged**